packages/tools/generator-widget/generators/app/templates/pluggable/native/fullTemplateJs/src/components/Badge.jsx.ejs

```diff
@@ -29,4 +29,4 @@ export class Badge extends Component {
 
         return text;
     }
-} 
+}
```

Comment:
Isnt the case that now we dont have the break line?

Reply:
There were two